py_swf/clients/workflow.py

```diff
@@ -104,11 +115,14 @@ def count_open_workflow_executions(
             latest_start_date=latest_start_date,
         )
 
-        return self.boto_client.count_open_workflow_executions(
-            domain=self.workflow_client_config.domain,
-            startTimeFilter=start_time_filter_dict['startTimeFilter'],
-            **workflow_filter_dict
+        count_open_workflows_result = CountWorkflowsResult(
+            **self.boto_client.count_open_workflow_executions(
+                domain=self.workflow_client_config.domain,
+                startTimeFilter=start_time_filter_dict['startTimeFilter'],
+                **workflow_filter_dict
+            )
         )
+        return count_open_workflows_result.count
```

Comment:
I think callers would benefit with the truncated result as well; this can be achieved by just returning the namedtuple, instead of just the count.